cmd/zetaclientd/start.go

```diff
@@ -205,7 +207,16 @@ func start(_ *cobra.Command, _ []string) error {
 
 	telemetryServer.SetIPAddress(cfg.PublicIP)
 	// Create TSS server
-	server, err := mc.SetupTSSServer(peers, priKey, preParams, appContext.Config(), tssKeyPass, true)
+	keygen := appContext.GetKeygen()
```

Comment:
NIT : the comment should be moved down to where we are creating the TSS server. 
We can add comment here to explain why we are creating the whitelist

Reply:
e772dd6774687135fc6b5a34df7a6361c3a7cd14
in this PR we don't need the whitelisted peers. 
The whitelisted peers is needed in conn gater #3028 
Removed here.